core/recoverer_test.go

```diff
@@ -28,6 +23,5 @@ func TestRecoverer(t *testing.T) {
 	if res != nil {
 		defer res.Body.Close()
 	}
-
-	require.True(t, strings.Contains(buffer.String(), "Panic: Test"))
+	require.Panics(t, nil, "Panic: Test")
```

Comment:
This doesn't look identical. The old test validated that something was logged. The new test panics because it's passed a `nil` function value.